test/TritonGPU/optimize_epilogue.mlir

```diff
@@ -0,0 +1,32 @@
+// RUN: triton-opt %s -split-input-file --tritonamdgpu-optimize-epilogue | FileCheck --check-prefixes=GCN %s
+
+#mma = #triton_gpu.mfma<{warpsPerCTA=[1,1], nonKDim = 32, isTranspose=false}>
```

Comment:
Can you change mma to mfma

Reply:
sure

FYI: current compiler generates `mma` as a base name for all "dot" layouts, including mfma.

Not sure if this change was intentional, though

Reply:
https://github.com/openai/triton/blob/main/lib/Dialect/TritonGPU/IR/Dialect.cpp#L2055